lib/tasks/javascript.rake

```diff
@@ -249,3 +269,28 @@ task 'javascript:update' do
 
   STDERR.puts "Completed copying dependencies: #{(Time.now - start).round(2)} secs"
 end
+
+task 'javascript:clean_up' do
```

Comment:
Suggestion: can we call this `javascript:cleanup`?

Reply:
I was leaning towards `javascript:cleanup`, but noticed we had `redis:clean_up` and `uploads:clean_up`. We don't appear to have tasks named `cleanup`, so I chose the existing popular choice.